app/views/user_mailer/issue_solved.html.haml

```diff
@@ -0,0 +1,5 @@
+Welcome, #{@member.name}
+%p An issue has just been solved.
+%p Website title: #{@website.title}
+%p #{@message}
+%p Thanks for joining and have a great day!
```

Comment:
Schimba mailu asta sa fie asa:

```
Hi, #{@member.name},

%p #{@message}

%p This email was sent because you subscribed to an error on our website.
```